security-analysis/security-analysis-api/src/main/java/com/powsybl/security/BusBreakerViolationLocation.java

```diff
@@ -7,35 +7,21 @@
  */
 package com.powsybl.security;
 
+import java.util.List;
 import java.util.Objects;
-import java.util.Optional;
 
 /**
  * @author Étienne Lesot {@literal <etienne.lesot at rte-france.com>}
  */
 public class BusBreakerViolationLocation implements ViolationLocation {
```

Comment:
We should write in the Java doc that it is Ids of configured buses of the bus/breaker view that are expected.